config/production.js

```diff
@@ -11,6 +11,13 @@ module.exports = {
   // mixpanel_token: '61e4b1fdd39e00551df8911fe62b8c56',
   luckyorange_enabled: true,
   pinterest: '0175a0c658a16a45e7c1f6b7cefaa34f',
+  db: {
+    sequelize: {
+      // The `url` property is documented in sequelize-cli readme but not in Sequelize core
+      url: process.env.DATABASE_URL,
```

Comment:
Oh okay, so to be clear, this URL includes all of our auth stuff in one neat variable? I love it!